<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelvizWindow</class>
 <widget class="QWidget" name="ChannelvizWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QChartView" name="plot"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ref</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Single Ended</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NRSE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diff</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PseudoDiff</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="suffix">
        <string> kS</string>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>Voltage</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Chunk</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>This is, currently, the only supported mode.</string>
       </property>
       <item>
        <property name="text">
         <string>Continuous</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>This is, currently, the only supported chunk size (INTERNAL_SAMP_SIZE).</string>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>ai0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ai1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ai2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ai3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ai4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ai5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ai6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ai7</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_9">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>No Connection</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
